lib/coinbase/wallet.rb

```diff
@@ -36,28 +47,69 @@ def import(data)
       end
 
       # Creates a new Wallet on the specified Network and generate a default address for it.
+      # @param interval_seconds [Integer] The interval at which to poll the CDPService for the Wallet to
+      # have an active seed, if using a ServerSigner, in seconds
+      # @param timeout_seconds [Integer] The maximum amount of time to wait for the ServerSigner to
+      # create a seed for the Wallet, in seconds
       # @param network_id [String] (Optional) the ID of the blockchain network. Defaults to 'base-sepolia'.
+      # @param server_signer [Boolean] (Optional) whether Wallet should use project's server signer. Defaults to false.
       # @return [Coinbase::Wallet] the new Wallet
-      def create(network_id: 'base-sepolia')
+      def create(network_id: 'base-sepolia', interval_seconds: 0.2, timeout_seconds: 20)
         model = Coinbase.call_api do
           wallets_api.create_wallet(
             create_wallet_request: {
               wallet: {
-                network_id: network_id
+                network_id: network_id,
+                use_server_signer: Coinbase.use_server_signer?
               }
             }
           )
         end
 
         wallet = new(model)
 
-        wallet.create_address
+        # Create a default address if the Wallet is not using the ServerSigner.
+        # When used with a ServerSigner, the Signer must first register
+        # with the Wallet before addresses can be created.
+        unless Coinbase.use_server_signer?
+          wallet.create_address
+          return wallet
+        end
 
+        # Wait until ServerSigner is active and create the default_address.
+        wait_for_signer(wallet.id, interval_seconds, timeout_seconds)
```

Comment:
This could take quite a while presumably? Do we want to have this block wallet creation on this?

It would be nice to always create a first address, but if I have a SDK method take 2-3+ seconds that's pretty brutal

Reply:
we could do a wait! that the developer has to explicitly call. Decided to go ahead with a blocking call for MVP